extensions/wikia/DesignSystem/bower.json

```diff
@@ -3,6 +3,6 @@
   "private": true,
 
   "dependencies": {
-    "design-system": "Wikia/design-system#0.1.1"
+    "design-system": "Wikia/design-system#XW-1575"
```

Comment:
We should change it after https://github.com/Wikia/design-system/pull/25 is merged and released.